assets/js/root.jsx

```diff
@@ -50,6 +44,11 @@ class Root extends React.Component {
         };
         this.channel.on("friend_request", friendRequestNotif.bind(this));
 
+        let pokeNotif = (payload) => {
+            this.setNotif(payload, "POKE");
```

Comment:
can we also add a case in the notification in `Notification` component? currently, only a friend request notif would show up because of the `switch/case` in it